README.md

```diff
@@ -180,9 +179,8 @@ make PG_CONFIG=/path/to/postgres/bin/pg_config install
 
 ```bash
 docker pull apache/age
-
 ```
-<h5> Create AGE docker container </h5>
+<h5> Create age docker container for first time</h5>
```

Comment:
You can capitalize the `age` word here as it holds significance

Reply:
the container name is case sensitive... Container won't run otherwise

Reply:
This is the title, affecting the case of the alphabet won't make any difference in the running of the code. The case-sensitive line is 193 specifying the image to pull.

Reply:
updated